requirements.txt

```diff
@@ -8,7 +8,7 @@ MarkupSafe==2.0.1
 semver==2.13.0
 chardet==3.0.4
 jsonpatch==1.33
-kubernetes==17.17.0
+kubernetes==30.1.0
```

Comment:
It's awesome you've bumped this one. I also noticed it but with some e2e failures due to api changes I abandoned it  to have it as part of other work.